src/subscription_manager/injectioninit.py

```diff
@@ -21,8 +21,8 @@
     EntitlementStatusCache, OverrideStatusCache, ProfileManager, \
     InstalledProductsManager, PoolTypeCache, ReleaseStatusCache, \
     RhsmIconCache, ContentAccessCache, PoolStatusCache, \
-    SyspurposeComplianceStatusCache, ContentAccessModeCache, \
-    SupportedResourcesCache, AvailableEntitlementsCache, \
+    SyspurposeComplianceStatusCache, SupportedResourcesCache, \
+    AvailableEntitlementsCache, ContentAccessModeCache, \
```

Comment:
This might be another harmless artefact of the cherry-pick. I'm tempted to say keep it unchanged unless necessary for some reason.

Reply:
yes, this was a result of the cherry-pick - I am leaving as is.